### PR TITLE
OPEN-15: Fix issue with modifiers being incorrectly mapped

### PR DESCRIPTION
**Pull request by teofilomonteiro**

### CODE REVIEW

**Comment by teofilomonteiro** [comment]
This solve the issue with the detecting the correct modifier of the methods, nevertheless, when you call `QueryClassWriter.buildMethodRefs` what is your intention? should synthetic methods also being included @anton-larsen-persequor ?

File: core/src/main/java/io/ran/MapperGenerator.java [hunk 1]
```diff
@@ -11,14 +11,20 @@
...
 			MappingClassWriter visitor = new MappingClassWriter(clazz.clazz);
 			byte[] bytes = visitor.toByteArray();
+			Files.write(path, bytes);
 			CheckClassAdapter.verify(new ClassReader(bytes),false, new PrintWriter(System.out));
 
 
 			QueryClassWriter visitor2 = new QueryClassWriter(clazz.clazz);
 			byte[] bytes2 = visitor2.toByteArray();
 			CheckClassAdapter.verify(new ClassReader(bytes2),false, new PrintWriter(System.out));
+			Files.write(pathQuery, bytes2);
```
Comment (anton-larsen-persequor) [inline core/src/main/java/io/ran/MapperGenerator.java:26]: This is for debug only, and should not be merged

Comment (teofilomonteiro) [reply core/src/main/java/io/ran/MapperGenerator.java:26]: I merge this 🤦 You are right!

**Review by anton-larsen-persequor — APPROVED** [review]
lgtm

**Pull request merged**